playbooks/installation/ldap_integration.adoc

```diff
@@ -1,16 +1,80 @@
+_________________________________________________
+During authentication, the LDAP directory is searched for an entry that matches the provided user name. If a single unique match is found, a simple bind is attempted using the distinguished name (DN) of the entry plus the provided password.
+_________________________________________________
+
+== LDAP Users and their Access to OpenShift
+
+Once it has been determined that access to OpenShift will be governed by users defined in an LDAP server, the next step is to determine the criteria for determining a valid user. Common use cases are listed below:
+
+* A user exists either at the same level or underneath a particular location of the LDAP tree (known as a base DN)
+* A user must be a member of a particular LDAP group
+
+NOTE: The conditions above assume the user has provided valid credentials
+
+=== Example LDAP Tree Structure
+
+The organization and structure of objects in an LDAP server is critical for its interaction with OpenShift. The following is an example of an LDAP structure that will be used for the rest of the discussion:
+
+[source,ldap]
+----
++ dc=example,dc=com # domain/organization
+  + dn: ou=Users,dc=example,dc=com 
+    + dn: uid=jdoe,ou=Users,dc=example,dc=com
+      cn: jdoe
+      givenName: John Doe
+      mail: jdoe@redhat.com
+      member: uid=admins,ou=Groups,dc=example,dc=com
+      member: uid=openshift,ou=Groups,dc=example,dc=com
+      phone: 919-555-1234
+      uid: jdoe
+    + dn: uid=csmith,ou=Users,dc=example,dc=com
+      cn: csmith
+      uid: csmith
+      mail: csmith@redhat.com
+      member: uid=openshift,ou=Groups,dc=example,dc=com
+      givenName: Chris Smith
+      phone: 919-555-4321
+  + dn: ou=Groups,dc=example,dc=com
+    + dn: uid=admins,ou=Groups,dc=sample,dc=com
+      member: uid=jdoe,ou=Users,dc=example,dc=com 
+    + dn: uid=openshift,ou=Groups,dc=example,dc=com
+      member: uid=jdoe,ou=Users,dc=example,dc=com
+      member: uid=csmith,ou=Users,dc=example,dc=com
+----
+
+To primary organizational units (OU) are defined within the `example.com` domain (`ou=Users,dc=example,dc=com` and `dn: ou=Groups,dc=sample,dc=com`). This example represents a flat hierarchy where all users within the organization are contained within the _Users_ OU and all groups are contained within the _Groups_ OU. 
+
+
+== Configure OpenShift to use LDAP for User Authentication
+
+To configure OpenShift to restrict access to the platform based on users managed in an LDAP server, an Identity Provider leveraging the `LDAPPasswordIdentityProvider` must be configured.  The following information must be available to configure LDAP based authentication:
+
+1. Username and password of a user that has access to traverse the LDAP tree (if anonymous access disabled)
+2. Protocol, hostname and port of the LDAP server
+3. CA bundle of the LDAP server (if secured)
```

Comment:
re: `(if secured)` -- I believe it should always be secured, or at least the default should be secure. ... so could we either word this differently or have a note about security somewhere?